queries.sql

```diff
@@ -68,3 +70,25 @@ WHERE date_of_birth
 BETWEEN '1990-01-01' 
 AND '2000-12-31' 
 GROUP BY species;
+
+-- Project 3
+--Join Query by owner name
+select * from animals JOIN owners on animals.owner_id=owners.id where owners.full_name='Melody Pond';
+
+--Join Query by species name
+select * from animals JOIN species on animals.species_id=species.id where species.name='Pokemon';
+
+--Full Join Query by owner
+select * from animals Full JOIN owners on animals.owner_id = owners.id;
```

Comment:
- [ ] Kindly remember to include those who don't own any animal, this is a requirement 👇🏽 

> List all owners and their animals, remember to include those that don't own any animal.

Reply:
corrected

![Screenshot (55)](https://github.com/Ademola-coding/vet_clinic/assets/96092850/853e162d-214d-4714-8cb8-c8cbdd6a7631)